README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
  - Check the `CONTRIBUTING.md` file for a guide on how to get started
  - This is a 100% volunteer-supported project, please be patient with your correspondence. We do handle issues and PRs with more fervor during Hacktoberfest & Conferences, but most (all?) of us have day jobs and so responses to questions / pending PRs may not be immediate. Please be patient, we'll get to you! :)
 
-Please feel free to join us on Slack! You can sign up at https://rubyforgood.herokuapp.com We're in #human-essentials
+#### 💬 Join us on slack
+Please feel free to join us on Slack!
+You can sign up at https://rubyforgood.herokuapp.com and find us in #human-essentials.
 
-The core team leads are: @albert @angela @sean @scott
 There are numerous other folks that can chime in and answer questions -- please ask and someone will probably be there to help!
 
-# README
-
-## About
+## 🛠️ Getting Started
 
-This application is an inventory management system that is built to address the needs of [Diaper Banks](https://nationaldiaperbanknetwork.org/diaper-need/) as directly and explicitly as possible, and later adapted to meet the need of other Essentials Banks.  Essentials Banks maintain inventory, receive donations and other means of intaking human essentials supplies (e.g. diapers, period supplies), and issue Distributions to community partner organizations. Like any non-profit, they also need to perform reports on this data, and have day-to-day operational information they need as well. This application aims to serve all those needs, as well as facilitate, wherever possible the general operations of the Diaper Bank themselves (eg. through using barcode readers, scale weighing, inventory audits).
+### Install Ruby
+You will need to first install the required ruby version specified in the `.ruby-version` file.  It is highly recommended you use a ruby version manager like [rbenv](https://github.com/rbenv/rbenv) or [asdf](https://asdf-vm.com/)
```

Comment:
Did a clean install of MacOS 12.4 with an Intel x86_64 MacBook. 
I had a problem after installing `rbenv` via `homebrew` where the shims folder was not setup. [This solved it](https://stackoverflow.com/questions/72140251/how-to-set-path-correctly-from-rbenv-shims-on-mac-m1-with-zsh)

Reply:
Did the section on installing ruby in GoRails below solve the issue?

Reply:
I didn't check the GoRails guide 🙈 
I'll take a look and share my findings